src/utils/async_stack_trace/src/lib.rs

```diff
@@ -235,20 +237,9 @@ pub trait StackTrace: Future + Sized {
 
     /// Similar to [`stack_trace`], but the span is a verbose one, which means it will be traced
     /// only if the verbose configuration is enabled.
-    #[cfg(not(debug_assertions))]
     fn verbose_stack_trace(self, span: impl Into<SpanValue>) -> StackTraced<Self, true> {
         StackTraced::new(self, span)
     }
-
-    /// Similar to [`stack_trace`], but the span is a verbose one, which means it will be traced
-    /// only if the verbose configuration is enabled.
-    ///
-    /// With `debug_assertions` on, this span will be disabled statically to avoid affecting
-    /// performance too much. Therefore, `verbose` mode in [`TraceConfig`] is ignored.
```

Comment:
Previously I thought the bad performance with `verbose` span, non-`verbose` configuration, and debug profile was caused by the extra code of matching the thread-local context. But actually, I just forgot to pass the config to the trace root and always use `verbose: true` configuration, as above. 🤣